include/eve/module/core/regular/impl/ifrexp.hpp

```diff
@@ -33,39 +34,97 @@
 
 namespace eve::detail
 {
-template<floating_ordered_value T>
-EVE_FORCEINLINE constexpr auto
-ifrexp_(EVE_SUPPORTS(cpu_), raw_type const&, T a0) noexcept
-
+template<typename T>
```

Comment:
you rewrote this file?

Reply:
It is just a migration of the pedantic case in a single file. Nothing more.